<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FMSSystem v0.1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(255, 255, 255);
	font-family: &quot;Microsoft Sans Serif&quot;;
}

QPushButton {
	border-radius: 0;
	border: 1px solid #000;
	min-height: 50px;
	min-width: 130px;
	margin: 10px;
	padding: 0 10px;
	font-size: 18pt;
}

QPushButton:hover {
	background-color: #eeeeee;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
       </font>
      </property>
      <widget class="QWidget" name="pageWellcome">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>99</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="labelWellcomeName">
             <property name="font">
              <font>
               <family>Microsoft Sans Serif</family>
               <pointsize>52</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="text">
              <string>FMSSystem</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelWellcomeDescription">
             <property name="font">
              <font>
               <family>Microsoft Sans Serif</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Система развития мелкой моторики</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>99</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="buttonWellcomeDemo">
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: #fff;
	background-color: #5cd25c;
}

QPushButton:hover {
	background-color: #5cb85c;
}</string>
             </property>
             <property name="text">
              <string>Демо</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonWellcomeExit">
             <property name="text">
              <string>Выход</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>99</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageMain">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>99</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="labelMainName">
             <property name="font">
              <font>
               <family>Microsoft Sans Serif</family>
               <pointsize>52</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="text">
              <string>FMSSystem</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelMainDescription">
             <property name="font">
              <font>
               <family>Microsoft Sans Serif</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Система развития мелкой моторики</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>99</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="buttonMainPlay">
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: #fff;
	background-color: #5cd25c;
}

QPushButton:hover {
	background-color: #5cb85c;
}</string>
             </property>
             <property name="text">
              <string>Играть</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonMainStats">
             <property name="text">
              <string>Статистика</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonMainSettings">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Настройки</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonMainExit">
             <property name="text">
              <string>Выход</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>99</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSettings">
       <widget class="QPushButton" name="buttonSettingsBack">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>500</y>
          <width>172</width>
          <height>72</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelSettingsName">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>136</y>
          <width>728</width>
          <height>59</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>52</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>Настройки</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageStats">
       <widget class="QPushButton" name="buttonStatsBack">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>500</y>
          <width>172</width>
          <height>72</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStatsName">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>136</y>
          <width>728</width>
          <height>59</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>52</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>Статистика</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageGame">
       <widget class="QPushButton" name="buttonGameBack">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>500</y>
          <width>172</width>
          <height>72</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelGameName">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>136</y>
          <width>728</width>
          <height>59</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>52</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>Игра</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
       <widget class="GameWidget" name="gameWidget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>800</width>
          <height>600</height>
         </rect>
        </property>
       </widget>
       <zorder>gameWidget</zorder>
       <zorder>buttonGameBack</zorder>
       <zorder>labelGameName</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GameWidget</class>
   <extends>QWidget</extends>
   <header location="global">game/gamewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
